static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -126,10 +140,14 @@ export function OnboardingLayout({
               organization={organization}
               platform={platformKey}
               projectId={projectId}
+              onClick={onProductSelectionClick}
             />
           )}
           {platformOptions && !['customMetricsOnboarding'].includes(configType) ? (
-            <PlatformOptionsControl platformOptions={platformOptions} />
+            <PlatformOptionsControl
+              platformOptions={platformOptions}
+              onClick={onPlatformOptionsClick}
```

Comment:
super nit: Why `onClick` vs `onChange`?
I see it like an input and there we would implement `onChange`, right? 🤔

Reply:
sure because we have an `input` element...in this case the `onChange` makes more sense